exporter/otlpexporter/otlp_test.go

```diff
@@ -277,7 +277,7 @@ func TestSendMetrics(t *testing.T) {
 	assert.EqualValues(t, 0, atomic.LoadInt32(&rcv.totalItems))
 
 	// A trace with 2 spans.
-	md = testdata.GenerateMetricsTwoMetrics()
+	md = testdata.GenerateMetricsTwoSumMetrics()
```

Comment:
Instead of adding new method maybe consider to change the other one and add same TODO.

Reply:
I thought adding another func would be less invasive, but it turns out I was wrong. Will add the same TODO to all the places I change here.

Reply:
@bogdandrutu I ended up adding 2 temporary test methods, changing the existing one caused all sorts of problems with other tests that were depending on them.

I plan on tackling the IntDataPoint/DoubleDataPoint conversion to NumberDataPoint soon, which will allow us to remove these temporary functions.